docs/go-quick-start.md

```diff
@@ -141,7 +141,8 @@ import (
 
 func main() {
 	// The client is a heavyweight object that should be created once
-	serviceClient, err := client.NewClient(client.Options{})
+	clientOptions := client.Options{HostPort: "localhost:7233"}
```

Comment:
Is there a Go sample that does this?
We will want to merge the Go sample into the doc page eventually - curious if it exists in the sample yet or if this is the only place? I checked a few samples but didn't see any -